src/wald_wolfowitz.jl

```diff
@@ -0,0 +1,68 @@
+export WaldWolfowitzTest, TwoValuedWaldWolfowitzTest
+
+struct WaldWolfowitzTest{T<:Real} <: HypothesisTest
+    nabove::Int     # Number of points above median (or of value a)
+    nbelow::Int     # Number of points below median (or of value b)
+    nruns::Int      # Number of runs
+    μ::T            # Expected mean
+    σ::T            # Expected variance
+    z::T            # test z-statistic from Normal distribution
+end
+
+testname(::WaldWolfowitzTest) = "Wald-Wolfowitz Test"
+population_param_of_interest(x::WaldWolfowitzTest) = ("Number of runs", x.μ, x.nruns) # parameter of interest: name, value under h0, point estimate
+default_tail(::WaldWolfowitzTest) = :both
+
+
+function show_params(io::IO, x::WaldWolfowitzTest, ident="")
+    println(io, ident, "number of runs:  $(x.nruns)")
+    println(io, ident, "z-statistic:     $(x.z)")
+end
+
+"""
+    WaldWolfowitzTest(x::AbstractVector{<:Real})
+
+Performs the Wald-Wolfowitz (or Runs) test of the null hypothesis that the given data is random, or independent.
```

Comment:
Just a matter of style, but can we really say that "data is independent"? The Wikipedia page says that *elements* are independent, which makes more sense to me.

Reply:
Does "independently sampled" seem more accurate?

Reply:
"elements are independently sampled" should be OK I guess.